features.yaml

```diff
@@ -343,7 +343,7 @@ features:
     level:
       checklist: features/ipv6-support.md
       maturity: Alpha
-      maturityNotes: Dual-stack IPv4 and IPv6 is not supported.
+      maturityNotes: Dual Stack IPv4 and IPv6 is not supported.
```

Comment:
This is confusing - the `maturityNotes` is an explaination for WHY its not more mature. So this should probably say "Dual stack is experimental still"

Reply:
Hi @howardjohn, okay, done!